### PR TITLE
IN-154: include healthcheck env vars in helm.

### PR DESCRIPTION
**Pull request by n3op2**
### What

Adding env vars to helm charts.

### CODE REVIEW

**Review by mattdean-digicatapult — APPROVED** [review]

File: helm/dscp-ipfs/templates/configmap.yaml [hunk 1]
```diff
@@ -5,6 +5,8 @@ metadata:
   labels:
     {{- include "dscp-ipfs.labels" . | nindent 4 }}
 data:
+  healthCheckPollPeriod: {{ .Values.config.healthCheckPort | quote }}
+  healthCheckTimeout: {{ .Values.config.healthCheckPort | quote }}
```
Comment (jonmattgray) [inline helm/dscp-ipfs/templates/configmap.yaml:9]: copy paste typo `Values.config.healthCheckPort`?

Comment (n3op2) [reply helm/dscp-ipfs/templates/configmap.yaml:9]: yes! trhanks!

File: helm/dscp-ipfs/templates/statefulset.yaml [hunk 2]
```diff
@@ -51,6 +51,16 @@ spec:
...
+            - name: HEALTHCHECK_POLL_PERIOD_MS
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ include "dscp-ipfs.fullname" . }}-config
+                  key: healthCheckPort
+            - name: HEALTHCHECK_TIMEOUT_MS
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ include "dscp-ipfs.fullname" . }}-config
+                  key: healthCheckPort
```
Comment (jonmattgray) [inline helm/dscp-ipfs/templates/statefulset.yaml:63]: copy paste typo, should be `healthCheckTimeout`

File: helm/dscp-ipfs/templates/statefulset.yaml [hunk 3]
```diff
@@ -51,6 +51,16 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy | quote }}
           command: [ './app/index.js' ]
           env:
+            - name: HEALTHCHECK_POLL_PERIOD_MS
+              valueFrom:
+                configMapKeyRef:
+                  name: {{ include "dscp-ipfs.fullname" . }}-config
+                  key: healthCheckPort
```
Comment (dblane-digicatapult) [inline helm/dscp-ipfs/templates/statefulset.yaml]: the key is incorrect, should be `healthCheckPollPeriod`

**Review by dblane-digicatapult — CHANGES_REQUESTED** [review]

**Review by jonmattgray — APPROVED** [review]

**Pull request merged**